src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -167,6 +203,7 @@ def main(
     # a cartesian product representation of the parameter space and
     # branches is a flat representation with the product expanded out.
     keyspace.persist(output_paths.keyspace, output_paths.branches)
+    total_num_jobs = len(keyspace)
```

Comment:
Nit: it's generally preferred to define variables as close as possible to their first usage unless there is some reason not to do so.